scipy/optimize/_qap.py

```diff
@@ -507,6 +519,29 @@ def _quadratic_assignment_faq(A, B,
     return OptimizeResult(res)
 
 
+def _adj_pad(A, B, method):
+    # pads the matrix with less nodes such that A & B are same size
```

Comment:
I'm confused by "less nodes"?

Reply:
would "fewer nodes" be more clear

Reply:
That would be the better word, but "fewer" relative to what?
Do you mean it pads the smaller of A and B to be the same sized as the larger?

Reply:
Yes. Your wording is more clear, so I'll replace what I had written with that.